table/metadata.go

```diff
@@ -163,10 +681,22 @@ type commonMetadata struct {
 	MetadataLog        []MetadataLogEntry      `json:"metadata-log"`
 	SortOrderList      []SortOrder             `json:"sort-orders"`
 	DefaultSortOrderID int                     `json:"default-sort-order-id"`
-	Refs               map[string]SnapshotRef  `json:"refs"`
+	SnapshotRefs       map[string]SnapshotRef  `json:"refs"`
 }
 
+func (c *commonMetadata) Ref() SnapshotRef                  { return c.SnapshotRefs[MainBranch] }
+func (c *commonMetadata) Refs() map[string]SnapshotRef      { return maps.Clone(c.SnapshotRefs) }
```

Comment:
lets use `maps.All` like i mentioned before for `slices.All`/`slices.Values` so that we can return an iterator without having to clone the whole map.